persistence/src/main/scala/hmda/persistence/institutions/SubmissionPersistence.scala

```diff
@@ -40,6 +41,12 @@ object SubmissionPersistence {
           }
 
           SubmissionState(submissions.updated(i, updatedSub))
+
+        case SubmissionFileNameAdded(id, name) =>
```

Comment:
We already have a `name` value defined in this scope, maybe change this to something else?

Reply:
Good call. Fixed in e2965f7.